### PR TITLE
Test Jenkins against multiple cluster versions on each PR

### PR DESCRIPTION
**Pull request by divyansh42**
Ref: https://issues.redhat.com/browse/JKNS-343

### CODE REVIEW

File: ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml [hunk 1]
```diff
@@ -65,6 +65,18 @@ releases:
       include_built_images: true
       name: "4.14"
       namespace: ocp
+  openshift-4_13:
```
Comment (divyansh42) [inline ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml:68]: @coreydaley please do let me know if you have any better suggestions for naming this.
It didn't allow me use `.` in name, so I used `_`

Comment (divyansh42) [reply ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml:68]: May be it can be `openshift_4_13` 🤔

**Comment by coreydaley** [comment]
@divyansh42 See if you can kick off the job rehearsals for the jenkins stuff and see if they work.

**Comment by coreydaley** [comment]
@divyansh42 Looks like some renaming is needed
 ERRO[2023-04-03T18:27:11Z] Some steps failed:                           
ERRO[2023-04-03T18:27:11Z] 
  * could not run steps: step [release:openshift-4_11] failed: failed to create or restart release pod: unable to create pod: Pod "release-openshift-4_11" is invalid: metadata.name: Invalid value: "release-openshift-4_11": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*') 
INFO[2023-04-03T18:27:11Z] Reporting job state 'failed' with reason 'executing_graph:step_failed:assembling_release:creating_release:running_pod' 

It's weird that none of the checks caught that ...

**Comment by coreydaley** [comment]
@divyansh42 The tests are not going to pass until you resolve the naming error that I pasted above.

**Comment by divyansh42** [comment]
> @divyansh42 The tests are not going to pass until you resolve the naming error that I pasted above.

Actually I updated at one type of failure but now it looks like it needs renaming at some other places too

**Comment by divyansh42** [comment]
Something weird with the tests, it is still showing older files

**Comment by coreydaley** [comment]
You may need to ask in #forum-testplatform on slack

**Comment by coreydaley** [comment]
/retest
infra issues bringing up a cluster

**Comment by coreydaley** [comment]
/hold
Once https://github.com/openshift/jenkins/pull/1666 is merged, and we get the new images released we will get back to seeing if we can get this to work, maybe we will try using hypershift instead.

**Comment by coreydaley** [comment]
/close
If we take another run at this we will start from scratch.

**Pull request closed without merging**